packages/editor/src/store/selectors.js

```diff
@@ -1748,3 +1748,21 @@ export function __experimentalGetTemplateInfo( state, template ) {
 		icon: templateIcon,
 	};
 }
+
+/**
+ * Returns a post type label depending on the current post.
+ *
+ * @param {Object} state Global application state.
+ *
+ * @return {string|null} The post type label if available, otherwise null.
+ */
+export const getPostTypeLabel = createRegistrySelector(
+	( select ) => ( state ) => {
+		const currentPostType = getCurrentPostType( state );
+		const postType = select( coreStore ).getPostType( currentPostType );
+		// Disable reason: Post type labels object is shaped like this.
+		// eslint-disable-next-line camelcase
+		return postType?.labels?.singular_name || null;
+		// translators: Default label for the Document sidebar tab, not selected.
```

Comment:
Forgot to remove 😆?

Reply:
> Forgot to remove 😆?

Let's keep this between you and me 😆 

P.S. Thanks for spotting!